index.js

```diff
@@ -43,42 +69,51 @@ async function run(config) {
 			}
 		}
 
-		if (typeof args.ignoreCerts !== 'undefined') {
+		if (config.ignoreCerts) {
 			process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
 		}
 
-		if (typeof args.schemas === 'string') {
-			let stat = await fs.lstat(args.schemas);
+		if (typeof config.schemas === 'string') {
+			let stat = await fs.lstat(config.schemas);
 			if (stat.isDirectory()) {
-				schemaFolder = normalizePath(args.schemas);
+				schemaFolder = normalizePath(config.schemas);
 			}
 			else {
 				throw new Error('Schema folder is not a valid STAC directory');
 			}
 		}
 
 		let schemaMapArgs = [];
-		if (Array.isArray(args.schemaMap)) {
-			// Recommended way
-			schemaMapArgs = args.schemaMap;
+		if (config.schemaMap && typeof config.schemaMap === 'object') {
+			if (Array.isArray(config.schemaMap)) {
+				// Recommended way via CLI
+				schemaMapArgs = config.schemaMap;
+			}
+			else {
+				// Recommended way via config file
+				schemaMapArgs = config.schemaMap;
```

Comment:
This is identical to the other branch, so I think you can collapse them.

Reply:
Indeed, that was some old code laying around ;-) Thanks.